index.html

```diff
@@ -223,14 +228,28 @@
         The term <dfn>webapp server</dfn> refers to server-side components of a <a>webapp</a>.
       </p>
       <p>
-        The term <dfn>push message</dfn> refers to an indication to a <a>webapp</a> that there is
-        new information for it from the <a>webapp server</a>, all or part of which MAY be contained
-        in the <a>push message</a> itself.
+        A <dfn>push message</dfn> is an indication to a <a>webapp</a> that there is new information
+        for it from the <a>webapp server</a>, all or part of which MAY be contained in the <a>push
+        message</a> itself.
+      </p>
+      <p>
+        A <dfn>push subscription</dfn> is a message delivery context that is established between
+        the <a>user agent</a> and the <a>push service</a> on behalf of a <a>service worker
+        registration</a>, and it is associated with that <a>service worker registration</a>. A
+        <a>push subscription</a> MUST NOT persist longer than its associated <a>service worker
+        registration</a>, but it MAY be terminated earlier.
+      </p>
+      <p>
+        A <a>push subscription</a> has an associated <dfn>endpoint</dfn>. It MUST be the absolute
+        URL exposed by the <a>push server</a> where the <a>webapp server</a> can send <a title=
+        "push message">push messages</a>. It MAY be the same for all <a title=
```

Comment:
This is a slightly rephrased section that used to accompany the IDL attribute description. I feel the rephrasing made it clearer and more correct. This change has nothing to do with the capability url discussion though. It seems both sides of that discussion can still conform to this section, due to the use of MAY, and it seems inter-operable as well.